Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -37,10 +38,21 @@ class MongoSwiftTestCase: XCTestCase {
     /// If the environment variable does not exist, this will use a default of "mongodb://127.0.0.1/".
     static var connStr: String {
         if let connStr = ProcessInfo.processInfo.environment["MONGODB_URI"] {
+            if topologyType == .sharded {
```

Comment:
nit: prefer explicit `self.topologyType` here (swiftlint rule for `explicit_self` seems to miss like... everything)